### PR TITLE
solve merge conflict for outlier report

### PR DESCRIPTION
**Pull request by Oysters1874**
Due to some merge conflicts and difficulties of resolving it in the original branch (generally, hard to merge dev into the feature branch as more conflicts occured) ( (https://github.com/NYCPlanning/data-engineering-qaqc/pull/77), I opened this new pr to solve the merge conflict.

### CODE REVIEW

File: src/pluto/helpers.py [hunk 1]
```diff
@@ -22,6 +22,9 @@ def get_data(branch) -> Dict[str, pd.DataFrame]:
     rv["df_expected"] = csv_from_DO(
         f"{url}/qaqc_expected.csv", kwargs={"converters": {"expected": json.loads}}
     )
+    rv["df_outlier"] = csv_from_DO(
+        f"{url}/qaqc_outlier.csv", kwargs={"converters": {"expected": json.loads}}
```
Comment (SashaWeinstein) [inline src/pluto/helpers.py:26]: Should be `"outlier"`

**Review by SashaWeinstein — COMMENTED** [review]
Really close! One functional change and the code should also be linted with bandit

File: src/pluto/helpers.py (hunk 1 above)
Comment (Oysters1874) [reply src/pluto/helpers.py:26]: oh sorry, I forgot this typo.

**Comment by Oysters1874** [comment]
> Really close! One functional change and the code should also be linted with bandit

ran it on my vscode, do we need to add any check for this on github?

**Comment by SashaWeinstein** [comment]
No check on github needed, I just noticed that when I saved it on my local machine (not in a container) some formatting changes were applied. Making sure it's fully linted now will avoid confusing change logs in the future

**Comment by SashaWeinstein** [comment]
My apologies, the formatting is done by [black](https://black.readthedocs.io/en/stable/), not bandit

**Comment by Oysters1874** [comment]
> My apologies, the formatting is done by [black](https://black.readthedocs.io/en/stable/), not bandit

okay cool, just done the reformatting

File: src/pluto/pluto.py [hunk 2]
```diff
@@ -20,12 +20,13 @@ def pluto():
 
     branches = get_branches()
     branch = st.sidebar.selectbox(
-        "select a branch",
-        branches,
-        index=branches.index("main"),
+        "select a branch", branches, index=branches.index("main"),
```
Comment (SashaWeinstein) [inline src/pluto/pluto.py:23]: Strange, when I run the black formatting it formats this chunk as 
```python
    branch = st.sidebar.selectbox(
        "select a branch",
        branches,
        index=branches.index("main"),
    )
```

**Review by SashaWeinstein — COMMENTED** [review]
The version of black on my machine formats `pluto.py` slightly differently but doesn't change `helpers.py` at all

File: src/pluto/pluto.py (hunk 2 above)
Comment (Oysters1874) [reply src/pluto/pluto.py:23]: not quite sure what causes the variation, is this one the only difference?

**Comment by SashaWeinstein** [comment]
On most recent version `black-22.6.0`

File: src/pluto/pluto.py (hunk 2 above)
Comment (SashaWeinstein) [reply src/pluto/pluto.py:23]: this change happens 3 times where it splits arguments across multiple lines. could it have to do with the version of black we are using?

**Comment by Oysters1874** [comment]
> black-22.6.0

oh my bad. I am running on an older version by just doing pip install. Update it now.

**Review by SashaWeinstein — APPROVED** [review]
Black says it's good, so I say it's good. Thanks for being patient while we worked through these merging and formatting issues! I'm heading straight home now that is this is done, hope you have a nice weekend and I'll see you Monday ☀️ !

File: src/pluto/pluto.py (hunk 2 above)
Comment (Oysters1874) [reply src/pluto/pluto.py]: yes, I guess it is due to the version. I update black to the latest version, now it seems look the same

**Pull request merged**